components/CopilotTooltip.tsx

```diff
@@ -0,0 +1,97 @@
+import React, {useContext} from 'react';
+import {useCopilot} from 'react-native-copilot';
+import {Text, Button, Row, Column} from './../components/ui';
+import {useTranslation} from 'react-i18next';
+import {GlobalContext} from '../shared/GlobalContext';
+import {AuthEvents, selectIsInitialDownload} from '../machines/auth';
+import {useSelector} from '@xstate/react';
+
+export const CopilotTooltip = () => {
+  const {t} = useTranslation('copilot');
+  const {
+    goToNext,
+    goToPrev,
+    stop,
+    currentStep,
+    isFirstStep,
+    isLastStep,
+    totalStepsNumber,
+    copilotEvents,
+  } = useCopilot();
+
+  const {appService} = useContext(GlobalContext);
```

Comment:
Can we move these logics to controller file? Similar to BannerNotificationController?